assets/queries/ansible/azure/container_registry_with_no_locks/query.rego

```diff
@@ -0,0 +1,39 @@
+package Cx
+
+CxPolicy [ result ]  {
+	document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+
+  	taskContainerRegistry := task["azure_rm_containerregistry"]
+
+	not xpto(taskContainerRegistry)
+
+
+		result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("name=%s.{{azure_rm_containerregistry}}.resource_group", [task.name]),
+                "issueType":		  "MissingAttribute",
```

Comment:
change issueType to IncorrectValue since we have the a lock, but it's an incorrect one